dj_cqrs/signals.py

```diff
@@ -73,6 +73,7 @@ def post_save(cls, sender, **kwargs):
                 instance.pk,
                 queue,
                 previous_data,
+                expires=get_expires_datetime(),
```

Comment:
I remember that we discussed that expiration may be set per model(instance), you will change that later?

Reply:
Expiration is per Service (maybe there was a misunderstanding).
We can add it easy later as model constant or static method.

For now it's looks little redundant.
Do you have business cases where this is useful per model?

Reply:
I don't (a the moment). But I don't have business cases, where it's needed per service. It has less sense, imho. So let's leave as is :)